pages/components/WizardExample.js

```diff
@@ -24,25 +23,14 @@ const WizardExample = () => {
   }
   return (
     <>
-      <Wizard onComplete={onComplete} stateManager={UrlStateManager} debug>
+      <Wizard
+        onReady={debounce(onReady)}
```

Comment:
Why do you need the debounce? :thinking:

Reply:
Removed commit

Reply:
Removed commit.